src/index.js

```diff
@@ -0,0 +1,88 @@
+// eslint-disable-next-line import/no-cycle
+import {
+  removeTask, editTask, addTask, clearFinishedTasks,
+} from './functions.js';
+import './style.css';
+import checkCompleted from './checkCompleted.js';
+
+const localStorageList = JSON.parse(localStorage.getItem('list'));
+
+const container = document.createElement('div');
+container.classList = 'todo-container';
+
+const listConainer = document.createElement('ul');
+listConainer.classList = 'list-container';
+listConainer.id = 'list-container';
+
+const clearBtn = document.createElement('button');
+clearBtn.classList = 'clear-btn';
+clearBtn.innerHTML = 'Clear all completed';
```

Comment:
Please keep the number of changes to the DOM as low as possible. Instead of creating one elements after the other. Why don't you try to use `innerHTML` to create a template for every list item.